### DISCUSSION TITLE
Migrate TimerX to Tukaan

### DISCUSSION THREAD

**Discussion by not-nef** [sumeshir26/TimerX#55] [Ideas]
Did you guys ever hear pf [tukaan](https://github.com/tukaan/tukaan)?

It seems to be a really nice framework but its still under development. It has some really cool features and i think we could make an app using it when the first stable release comes out

**Comment by lg-nineoneninethreenine** [comment]
It's on my watchlist! Also what is "pf"?

**Reply by lg-nineoneninethreenine** [reply]
Is it "of"? More possible since P and O are near each other in the QWERTY keyboard.

**Reply by not-nef** [reply]
> It's on my watchlist! Also what is "pf"?

A typo lmao

**Reply by not-nef** [reply]
Happens to me a lot

**Reply by lg-nineoneninethreenine** [reply]
Try using Grammarly, that's a good tool in the free spectrum.

**Comment by sumeshir26** [comment]
@not-nef I literally fully watch that repo I was going to do the same thing! Even TimerX can possibly be upgraded to it once it goes on PIP

**Comment by lg-nineoneninethreenine** [comment]
Moving this to ideas since this is an idea.

**Comment by not-nef** [comment]
> @not-nef I literally fully watch that repo I was going to do the same thing! Even TimerX can possibly be upgraded to it once it goes on PIP

Epic

**Comment by not-nef** [comment]
I just stumbled over it when looking for more sun valley ttk stuff. It's literally such a cool project i think we should definitely convert TimerX to tukaan. (As soon as its stable)

**Comment by lg-nineoneninethreenine** [comment]
We could use Tukaan and a bit of GTK for Python. AFAIK, in the Linux universe, half of them use GNOME. The rest use KDE or LXDE, Xfce, etc. We could try adding a bit of GTK for Linux users since Tukaan is native to Windows and macOS. SO in that process, it's native to all three OSes(at least GNOME-based distros in Linux though, KDE is a pain I guess).

**Reply by sumeshir26** [reply]
rdbende had a lot of nice ttk themes over [here](https://rdbende.github.io/tkinter-docs/resources/list-of-ttk-themes.html). We could use Radiance for Linux

**Reply by lg-nineoneninethreenine** [reply]
That theming seems like GTK2 or Adwaita 2018. I want it to kinda match GTK3.
